src/app/modules/angular-slickgrid/styles/_variables.scss

```diff
@@ -145,6 +145,12 @@ $autocomplete-border-radius:	                  4px !default;
 $autocomplete-box-shadow:	                      0 6px 12px rgba(0, 0, 0, 0.175) !default;
 $autocomplete-hover-color:	                    #262626 !default;
 $autocomplete-hover-bg-color:	                  darken($row-mouse-hover-color, 3%) !default;
+$autocomplete-loading-icon:                     "\f021" !default;
+$autocomplete-loading-icon-color:               inherit !default;
```

Comment:
use `$icon-color` instead and also do the same for the var `$header-menu-button-icon-color`